mula/scheduler/schedulers/scheduler.py

```diff
@@ -127,10 +141,20 @@ def post_pop(self, p_item: models.PrioritizedItem) -> None:
             )
             return None
 
+        # Update task
         task.status = models.TaskStatus.DISPATCHED
         self.ctx.datastores.task_store.update_task(task)
 
-        return None
+        # Create event
+        event = models.Event(
+            task_id=task.id,
+            type="events.db",
+            context="task",
+            event="update",
+            data=task.model_dump(),
```

Comment:
This will result in a very big event table, because the task model is pretty big already and this will be duplicated unnecessarily multiple times in the events table. I don't think this is a good idea with regards to performance and resource usage.